<table align="left">
  <td>
    <a href="https://colab.research.google.com/github/marco-canas/4_didactica_aritmetica/blob/main/modelo_general_clase_aula_invertida.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
  </td>
</table>

### [Video de apoyo a la lectura interactiva y experimental de este cuaderno]()

## [Vínculo al programa del curso:  ]()


Los estudiantes son la razón de nuestro trabajo: 


<img src = 'https://github.com/marco-canas/1_algebra/blob/main/classes/ciencia_datos/8_class_april_19_regla%20de%20tres_inversa_quiz_regla_tres_directa/trabajo_equipo.jpg?raw=true'>

# Modelo general de clase: [aula invertida](https://www3.gobiernodecanarias.org/medusa/ecoescuela/pedagotic/aprendizaje-invertido-flipped-classroom/)

## Definición y características de este modelo

## ¿Qué es?  

Flipped Learning o aprendizaje invertido es un enfoque pedagógico en el que:  

* la instrucción directa se realiza fuera del aula y 
* se utiliza el tiempo de clase para llevar a cabo actividades que impliquen el desarrollo de procesos cognitivos de mayor complejidad, en las que son necesarias la ayuda y la experiencia del docente.

## ¿Para qué sirve?  

Mejora el proceso de enseñanza-aprendizaje realizando,   

* fuera del aula, actividades de aprendizaje sencillas (observar, memorizar, resumir, etc.) y, 
* en el aula, actividades más complejas (razonar, examinar, priorizar, argumentar, proponer, etc.) que requieren la interacción entre iguales y la ayuda del docente como facilitador.

<img src = 'https://github.com/marco-canas/7_didactica_ciencia_datos/blob/main/propuesta/unidad_didac/atributos_aprendizaje_invertido.png?raw=true'>

## Fomento de la lectura literaria asociada a Ingeniería agropecuaria y licenciatura en Matemáticas

John Allen Paulos. El hombre anumérico. El analfabetismo matemático y sus consecuencias. 

"Las mates siempre fueron mi asignatura más floja." 



### Objetivo de aprendizaje

* Aprenda a utilizar la proporcionalidad inversa en situaciones de ingeniería agropecuaria en Caucasia. 

## Establecimiento de prerrequisitos conceptuales para alcanzar el objetivo de aprendizaje trazado. 
1. regla de tres directa y ejemplos. 
2. 
3. 

# Para el diseño de la clase tenga en cuenta algunos problemas de aprendizaje

Asume que en todas las relaciones crecientes aplica regla de tres 

Ejemplo: https://www.google.com/search?client=firefox-b-d&q=convertir+fahrenheit+a+celsius
        
        la relación entre grados celsius y fahrenheit es creciente pero no es de proporcionalidad directa. 
        
[termometros en paralelo](https://espanol.libretexts.org/Fisica/Libro%3A_F%C3%ADsica_(sin_l%C3%ADmites)/12%3A_Temperatura_y_Teor%C3%ADa_Cin%C3%A9tica/12.2%3A_Escalas_de_Temperatura_y_Temperatura#:~:text=Las%20escalas%20Fahrenheit%20y%20Celsius,define%20como%20%2D459.67%20%C2%B0F)

## Tereas para realizar antes de la clase

1. Ver los videos respectivos []()
2. Memorizar las definiciones [regla de tres inversa](https://udeaeduco-my.sharepoint.com/:u:/g/personal/marco_canas_udea_edu_co/EfADcdsqzZZGul6O8jyiDtEBOkH5kbgiQfWUaObgBebSAg?e=Q7hfHv)
3. Sintetizar o resumir la información []()

# Tareas para realizar en clase

* Modelación matemática 
* Trabajo en equipos

# Problema real de proporcionalidad inversa para ingeniería agropecuaria

[Precálculo de stewart](https://udeaeduco-my.sharepoint.com/:b:/g/personal/marco_canas_udea_edu_co/ERHIq62I6qFNrmxy6LZb8ZMBci7kUsyNME1nIh9yCBMJ_w?e=od3T7I)

Página 133 del PDF.



## EJEMPLO 2 Variación inversa

La Ley de Boyle dice que cuando una muestra de gas se comprime a una temperatura constante, **la presión del gas es inversamente proporcional al volumen** del gas.

(a) Suponga que la presión de una muestra de aire que ocupa $0.106 m^{3}$
 a 25ºC es 50 kPa. 
Encuentre la constante de proporcionalidad y escriba la ecuación que expresa la proporcionalidad inversa.  

$$ P = \frac{K}{V}  $$

$$50000Pa = \frac{K}{0.106m^{3}}$$

$$ 50000 (0.106) = K  $$

In [3]:
50*0.106

5.3

$$k = 5.3$$

$$ P = \frac{5.3}{V} $$

este es el modelo adaptado. 

(b) Si la muestra se expande a un volumen de 0.3 m3, encuentre la nueva presión.

Mis estudiantes son la razón de mi actividad docente:

<img src = 'https://github.com/marco-canas/2_calculo_I/blob/main/classes/7_class_april_14_differentiation/IMG-20221102-WA0046.jpeg?raw=true'>

# Tareas para realizar después de la realización de la clase

* Actividades de memorización: 
  - escucha reiterada de grabaciones de formulas y resultados científicos. 
  - relectura de notas. 
  - ejercicios de mayor complejidad. 
¿Cuanto vale un metro cúbico de agua en Aguascol Caucasia? 
¿Cuantos meteros cubicos se consumen en su casa por mes?
Enviar foto del metro cúbico construido por los estudiantes de agropecuaria

1. 

## Dificultades de aprendizaje detectados durante la clase

## Estrategias propuestas para la solución de las dificultades de aprendizaje detectadas

## Cuestionario para realizar labores de memorización y observación,  a la síntesis de la información

## Grabaciones en audio para fomentar la capacidad de memorizar enunciados de teoremas y resultados científicos y la capacidad de parafrasear estos resultados. 

Se dispondrá de un lugar en el drive asociuado al correo institucional de la Universidad Nacional del Profesor Marso para albergar los archivosd de audio de fomento a la memoirización, enunciación y parafraseo de los resultados científicos, prtopiedades, reglas, etc. 

* Vocaroo

### Comparta sus cuadernos y videos, evidencias de su aprendizaje [aquí](https://forms.office.com/Pages/ResponsePage.aspx?id=IefhmYRxjkmK_7KtTlPBwkanXIs1i1FEujpsZgO6dXpUMEMwTEZVMzEwVVBPTElWNVg5OVEyWUhMUy4u)

# Vocabulary 

## Mapa conceptual que vincula los conceptos asoaciados al objetivo de aprendizaje

## Quiz de 5 minutos para ejercitación y adquisición de habilidades 

## Referentes 

* [stewart precálculo]()
* [stewart cálculo]()
* [larson]()

* [uzcategui](https://udeaeduco-my.sharepoint.com/:b:/g/personal/marco_canas_udea_edu_co/ETDikm-lVl1Or8XoEo9oyh0BEti9Zs8le-f0D-dBdtZmbA?e=bBsoyQ)

### [Evaluamos al profesor Marco Cañas Aquí](https://forms.office.com/Pages/ResponsePage.aspx?id=IefhmYRxjkmK_7KtTlPBwkanXIs1i1FEujpsZgO6dXpUREJPV1kxUk1JV1ozTFJIQVNIQjY5WEY3US4u)

## [Evaluación luego de alcanzar estos objetivos de aprendizaje]()

### Continue su aprendizaje en la siguiente clase a través del siguiente [vínculo]()

# Conjeturas pedagógicas fruto de la aplicación del modelo de aprendizaje invertido y del enfoque hacia la ciencia de datos con python

1. Todo cálculo o resultado debe ser interpretado en una línea markdown en la línea del cuaderno Jupyter, inmediatamente después de la enunciación del resultado y después de la presentación de una tabla o gráfico bidimensional, de tal menera que el estudiante explicite la comprensión verbal del resultado y las inferencias o estrategias que este resultado le sugieren.   


### Agradecimientos  

Doy gracias a Dios por la vida de mi Hijo Joseph Cañas Osorio y la madurez que ha alcanzado. 